src/components/PageDefinitions.js

```diff
@@ -192,18 +197,18 @@ class PageDefinitions extends Component {
 
   renderContributeButton() {
     return (
-      <div>
-        <Button
-          bsStyle="success"
-          className="pull-right"
-          disabled={!this.hasChanges()}
-          onClick={this.doPromptContribute}
-        >
-          Contribute
+      <div className="pull-right">
+        <Button bsStyle="danger" disabled={!this.hasComponents()} onClick={this.onRemoveAll}>
+          Remove All
         </Button>
+        &nbsp;
```

Comment:
ok for now but should be using CSS for spacing etc rather than &nbsp;. I know it is used elsewhere. Laziness.